integration_tests/src/test/kotlin/utils/CommandHelper.kt

```diff
@@ -3,11 +3,33 @@ package utils
 import java.io.File
 import java.util.concurrent.TimeUnit
 
+
 fun String.runCommand(workingDir: File): ProcessResult = ProcessBuilder(*split("\\s".toRegex()).toTypedArray())
-    .directory(workingDir)
-    .redirectOutput(ProcessBuilder.Redirect.PIPE)
-    .redirectError(ProcessBuilder.Redirect.PIPE)
-    .start().run {
-        waitFor(60, TimeUnit.MINUTES)
-        ProcessResult(exitValue(), inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText())
+    .directory(workingDir).run {
+        redirectOutputForCompatibleOS().start().run {
+            waitFor(processTimeout, TimeUnit.MINUTES)
+            ProcessResult(exitValue(), getOsSpecificOutput())
+        }
+    }
+
+private fun Process.getOsSpecificOutput(): String {
+    return if (isWindows) {
+        File(outputFileName).readText() + File(errorFileName).readText()
+    } else {
+        inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText()
     }
+}
+
+private fun ProcessBuilder.redirectOutputForCompatibleOS(): ProcessBuilder {
+    return if (isWindows) {
+        redirectOutput(File(outputFileName)).redirectError(File(errorFileName))
+    } else {
+        redirectOutput(ProcessBuilder.Redirect.PIPE).redirectError(ProcessBuilder.Redirect.PIPE)
+    }
+}
```

Comment:
we could skip type and `return` for both cases

Reply:
:)

Reply:
still not fixed I asked for replacing
```kotlin
private fun ProcessBuilder.redirectOutputForCompatibleOS(): ProcessBuilder {
    return if (isWindows) {
        redirectOutput(File(outputFileName)).redirectError(File(errorFileName))
    } else {
        redirectOutput(ProcessBuilder.Redirect.PIPE).redirectError(ProcessBuilder.Redirect.PIPE)
    }
}
```
to 
```kotlin
private fun ProcessBuilder.redirectOutputForCompatibleOS() = if (isWindows) {
        redirectOutput(File(outputFileName)).redirectError(File(errorFileName))
    } else {
        redirectOutput(ProcessBuilder.Redirect.PIPE).redirectError(ProcessBuilder.Redirect.PIPE)
    }
```

and same for `getOsSpecificOutput`

Reply:
If you prefer the current version just leave it :)

Reply:
fixed 👍